packages/kit/src/runtime/server/index.js

```diff
@@ -289,7 +290,7 @@ export async function respond(request, options, state) {
 								}
 							});
 
-							response = json({
+							return json({
```

Comment:
uh i think this change was made accidentally. would result in cookies not being set etc

```suggestion
							response = json({
```